libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -820,6 +820,8 @@ Vector3f AP_AHRS::wind_estimate(void) const
 bool AP_AHRS::airspeed_estimate(float &airspeed_ret) const
 {
     bool ret = false;
+
+#if AP_AIRSPEED_ENABLED && !(APM_BUILD_TYPE(APM_BUILD_ArduCopter) && BOARD_FLASH_SIZE < 1025)
```

Comment:
why the extra checks in the #if ?

Reply:
As said on call: these are necessary to define out Airspeed for copters with 1MB boards. APM_BUILD_TYPE is not propagated to all of the .cpp and .h and thus this cannot be added to AP_AIRSPEED.h

Reply:
Using the Dummy class now